css/amp-story-player.css

```diff
@@ -65,3 +65,14 @@ amp-story-player::after {
 .i-amphtml-story-player-loaded::after {
   visibility: hidden;
 }
+
+/** Apply click shield on the player if it is on autoplay mode. */
+.i-amphtml-story-player-loaded[autoplay]::after {
+  all: initial;
```

Comment:
Not needed?

Reply:
It is, since there are styles applied to that `::after` element previously, including an animation that makes it spin 😵